src/semantic-highlighting.ts

```diff
@@ -292,7 +292,7 @@ export class ThemeRuleMatcher {
   // Returns the best rule for a scope.
   getBestThemeRule(scope: string): TokenColorRule {
     if (this.bestRuleCache.has(scope))
-      return this.bestRuleCache.get(scope);
+      return this.bestRuleCache.get(scope)!;
```

Comment:
I would consider this a safe non-null assertion, since we have already checked that the `Map` has the value before getting it.